library/src/main/java/io/ona/kujaku/views/KujakuMapView.java

```diff
@@ -1211,6 +1213,21 @@ public void onMapReady(MapboxMap mapboxMap) {
         }
     }
 
+    @Override
+    public boolean changeLocationUpdates(long updateInterval, long fastestUpdateInterval, int accuracyLevel) {
+        if (getLocationClient() != null) {
```

Comment:
Should there be validation for the arguments in this function?

I think it's possible to give invalid ones e.g. negatives.